desc/compute/_neoclassical.py

```diff
@@ -0,0 +1,261 @@
+"""Compute functions for neoclassical transport.
+
+Notes
+-----
+Some quantities require additional work to compute at the magnetic axis.
+A Python lambda function is used to lazily compute the magnetic axis limits
+of these quantities. These lambda functions are evaluated only when the
+computational grid has a node on the magnetic axis to avoid potentially
+expensive computations.
+"""
+
+from functools import partial
+
+from orthax.legendre import leggauss
```

Comment:
is it important for this to be in JAX? or if all it is doing is getting the points to evaluate at, and num_quad is not beind differentiated, then yea we can just use scipy

Reply:
I refactored so that we compute the quadrature points and weights once inside objective.build rather than  every objective.compute. However, if the user is just calling `eq.compute("effective ripple", grid=grid)`, then I think we need JAX since the compute function is JITed.